src/renderer/renderer.ts

```diff
@@ -711,38 +711,41 @@ export class SDFGRenderer {
 
             // Filter graph to selection (visual cutout).
             this.cutoutBtn = $('<button>', {
+                id: 'cutout-button',
                 class: 'btn btn-light btn-sdfv-light btn-sdfv hidden',
                 html: '<i class="material-icons">content_cut</i>',
                 title: 'Filter selection (cutout)',
                 click: () => {
                     this.cutout_selection();
                 },
             }).appendTo(this.toolbar);
-            this.cutoutBtn.hide();
+            this.cutoutBtn.addClass('hidden');
 
             // Transition to local view with selection.
             this.localViewBtn = $('<button>', {
+                id: 'local-view-button',
                 class: 'btn btn-light btn-sdfv-light btn-sdfv hidden',
                 html: '<i class="material-icons">memory</i>',
                 title: 'Inspect access patterns (local view)',
                 click: () => {
                     this.localViewSelection();
                 },
             }).appendTo(this.toolbar);
-            this.localViewBtn.hide();
+            this.localViewBtn.addClass('hidden');
 
             // Exit previewing mode.
             if (this.in_vscode) {
                 const exitPreviewBtn = $('<button>', {
+                    id: 'exit-preview-button',
                     class: 'btn btn-light btn-sdfv-light btn-sdfv hidden',
                     html: '<i class="material-icons">close</i>',
-                    title: 'Exit preview',
+                    title: 'Exit previewyyy',
```

Comment:
:)

Reply:
oops, a sanity check for detecting changes.. :-)